### ISSUE TITLE
US Inventory? 

### ISSUE THREAD

**Issue by ferociouskyle** [teslahunt/inventory#3]
Perhaps I am not seeing the option to use US as the inventory. But see as a few other countries are listed, I assume that US id default or is US not available?

**Comment by Kikobeats** [comment]
Hello, right now only Europe inventory is supported.

It should be easy to add USA inventory, make can you convert this into a PR? 🙂

**Comment by Kikobeats** [comment]
published at `tesla-inventory@1.3.0`:

```js
const teslaInventory = require('tesla-inventory')

const results = await teslaInventory('usa', {
    condition: 'used',
    model: 'y'
  })
```
